queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),
+	}
+	return &q
+}
+
+func (q *Queue) AddSong(songID string) error {
+	//TODO: check if song already added
+	q.songList = append(q.songList, songID)
+	return nil
+}
+
+func (q *Queue) RemoveSong(songID string) error {

```

Comment:
If you want to return an error here we should check for the song not being in the queue